modules/populate-registry-restricted-network-cluster.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+[id="populate-registry-restricted-network-cluster_{context}"]
+= Populate the registry
+
+From within your disconnected environment, tag and push the images to your internal registry:
+
+[IMPORTANT]
+====
+The following steps are a generic guide to loading the images into a registry.
+You might need to take more or different actions to load the images.
+====
+
+. Before you push the images into the registry, re-tag each image.
+** For images in the `openshift3` repository, tag the image as both the major and minor version number. For example, to tag the {product-title} node image:
```

Comment:
@jianlinliu - Should these be `openshift4` repositories?

Reply:
modules/installation-mirror-repository.adoc already mirrorred images when upgrading rhcos cluster, not sure why we need this module here.